backend/coins/btc/account.go

```diff
@@ -364,30 +367,33 @@ func (account *Account) Initialize() error {
 		account.coin.Net(), account.db, theHeaders, account.synchronizer,
 		account.blockchain, account.notifier, account.log)
 
-	gapLimits, err := account.gapLimits(signingConfiguration)
+	// TODO: unified-accounts
+	gapLimits, err := account.gapLimits(signingConfigurations[0])
 	if err != nil {
 		return err
 	}
 
 	account.log.Infof("gap limits: receive=%d, change=%d", gapLimits.Receive, gapLimits.Change)
 
-	var subacc subaccount
-	subacc.signingConfiguration = signingConfiguration
-	if signingConfiguration.IsAddressBased() {
-		subacc.receiveAddresses = addresses.NewSingleAddress(
-			signingConfiguration, account.coin.Net(), account.log)
-		account.log.Debug("creating single change address for address based account")
-		subacc.changeAddresses = addresses.NewSingleAddress(
-			signingConfiguration, account.coin.Net(), account.log)
-	} else {
-		subacc.receiveAddresses = addresses.NewAddressChain(
-			signingConfiguration, account.coin.Net(), int(gapLimits.Receive), 0, account.log)
-		subacc.changeAddresses = addresses.NewAddressChain(
-			signingConfiguration, account.coin.Net(), int(gapLimits.Change), 1, account.log)
+	for _, signingConfiguration := range signingConfigurations {
+		signingConfiguration := signingConfiguration
+
+		var subacc subaccount
+		subacc.signingConfiguration = signingConfiguration
+		if signingConfiguration.IsAddressBased() {
+			subacc.receiveAddresses = addresses.NewSingleAddress(
+				signingConfiguration, account.coin.Net(), account.log)
+			account.log.Debug("creating single change address for address based account")
+			subacc.changeAddresses = addresses.NewSingleAddress(
+				signingConfiguration, account.coin.Net(), account.log)
+		} else {
+			subacc.receiveAddresses = addresses.NewAddressChain(
+				signingConfiguration, account.coin.Net(), int(gapLimits.Receive), 0, account.log)
+			subacc.changeAddresses = addresses.NewAddressChain(
+				signingConfiguration, account.coin.Net(), int(gapLimits.Change), 1, account.log)
+		}
+		account.subaccounts = append(account.subaccounts, subacc)
 	}
-	// TODO: unified-accounts
```

Comment:
fixed the first todo, rest already in the backlog ;)

Reply:
:+1: